### PR TITLE
start moving global styles to styleguide

### PR DESCRIPTION
**Pull request by sandikbarr**
requires https://github.com/PRX/styleguide.prx.org/pull/20

### CODE REVIEW

File: .angular-cli.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
         "../node_modules/pikaday/css/pikaday.css",
         "../node_modules/pikaday/css/triangle.css",
         "../node_modules/c3/c3.css",
-        "styles.css"
+        "styles.css",
+        "../node_modules/ngx-prx-styleguide/styles.css"
```
Comment (sandikbarr) [inline .angular-cli.json]: normally I would want to use the styleguide before the app styles.css, but reset is doing all kinds of things to us that need to be cleaned up before we can override styleguide styles with app specifics

File: src/assets/styles/app.css [hunk 2]
```diff
@@ -1,103 +0,0 @@
-/*
- * Fonts
- */
-@import url(https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,700italic,300,400,700);
-
-/*
- * Media
- */
```
Comment (sandikbarr) [inline src/assets/styles/app.css]: `.media` and `.img` are not in use and removed from https://github.com/PRX/styleguide.prx.org/pull/20/files#diff-478c8efeb2d9a5f992f435f5dcd2afb5R1

File: src/assets/styles/app.css [hunk 3]
```diff
@@ -1,103 +0,0 @@
...
-.media .img {
-  float: left;
-  margin-right: 10px;
-}
-
-/*
- * Meta
- */
```
Comment (sandikbarr) [inline src/assets/styles/app.css]: .meta is not in use and is removed from https://github.com/PRX/styleguide.prx.org/pull/20/files#diff-478c8efeb2d9a5f992f435f5dcd2afb5R1

File: src/assets/styles/app.css [hunk 4]
```diff
@@ -1,103 +0,0 @@
...
-  line-height: 17px;
-  color: #fff;
-}
-a.button:hover, a.button:active, a.button:focus {
-  color: #fff;
-}
-
-/*
- * Dragula
- */
```
Comment (sandikbarr) [inline src/assets/styles/app.css]: Dragula moved to https://github.com/PRX/publish.prx.org/pull/411/files#diff-b133c7c05994ecce85295092da6ef47dR1

File: src/assets/styles/layout.css [hunk 5]
```diff
@@ -1,234 +0,0 @@
-article .hero > section,
-article .hero div > section,
-article .main > section,
-article .asides aside > section {
```
Comment (sandikbarr) [inline src/assets/styles/layout.css]: .asides is not in use in publish, so I have not moved it into the styleguide https://github.com/PRX/styleguide.prx.org/pull/20/files#diff-5b8e63d7964be0aaeb36dee44f1e1ca3R38 Looks like it came from beta.

File: src/assets/styles/layout.css [hunk 6]
```diff
@@ -1,234 +0,0 @@
...
-  background: #e6e6e6;
-  position: relative;
-}
-@media multi-column {
-  .empty-space {
-    min-height: 420px;
-  }
-}
-@media screen and (min-width: 768px) {
-  article .content {
```
Comment (sandikbarr) [inline src/assets/styles/layout.css]: `.content` doesn't exist in publish. Looks like this came from beta. In Publish, we're giving the .main container a bunch of flex properties but it's not inside a `display: flex` container. Looks like it should be in this .content container. https://github.com/PRX/styleguide.prx.org/pull/20/files#diff-5b8e63d7964be0aaeb36dee44f1e1ca3R60

File: src/assets/styles/layout.css [hunk 7]
```diff
@@ -1,234 +0,0 @@
...
-a:active {
-  color: #2b7187;
-}
-a.current:hover {
-  text-decoration: none;
-}
-strong {
-  font-weight: bold;
-}
-.unbreakable {
```
Comment (sandikbarr) [inline src/assets/styles/layout.css]: `.unbreakable` and `.empty-space` don't exist in publish

File: src/assets/styles/layout.css [hunk 8]
```diff
@@ -1,234 +0,0 @@
...
-  padding-top: 50px;
-  position: relative;
-}
-@media screen and (min-width: 768px) {
-  main {
-    min-height: 400px;
-    padding-top: 73px;
-  }
-}
-h1,
```
Comment (sandikbarr) [inline src/assets/styles/layout.css]: moved to app.less https://github.com/PRX/styleguide.prx.org/pull/20/files#diff-478c8efeb2d9a5f992f435f5dcd2afb5R6

File: src/assets/styles/layout.css [hunk 9]
```diff
@@ -1,234 +0,0 @@
...
-    -webkit-flex-shrink: 0;
-    flex-shrink: 0;
-    flex-basis: 280px;
-    -ms-flex-preferred-size: 280px;
-    -webkit-flex-basis: 280px;
-    max-width: 280px;
-    margin-right: 20px;
-  }
-}
-article .content .gallery > header {
```
Comment (sandikbarr) [inline src/assets/styles/layout.css]: `.gallery` doesn't exist in publish

**Comment by sandikbarr** [comment]
I will update the package.json with a new ngx-prx-styleguide version once it has been published from https://github.com/PRX/styleguide.prx.org/pull/20.

**Comment by gcampo88** [comment]
Currently comparing publish.prx.dev (with styleguide #20 changes included manually in node_modules) against publish.prx.org, and noticed a couple things visually-- here's a screenshot (left is dev, right is prod) 
image

- looks like some places that are #676767 (`grey-dove`) in production (e.g., the last-saved info in the hero, the `fancy-hint` divs on the basic info, `<dt>`s in the status box) are #818181  (`grey`) in dev
- dev design is much more [gracefully] responsive to small screen size! maybe that .content vs .main thing you found? (just saw the headline about that, have to look more closely).  👏 


now will start looking at code!

**Comment by sandikbarr** [comment]
Latest commit to styleguide should fix that grey vs grey-dove on the body. I went back and forth with that color naming so much! Really good catch. https://github.com/PRX/styleguide.prx.org/pull/20/commits/87a3fddefa912463c750044bb58b871fd1b28d73

File: src/assets/styles/app.css (hunk 2 above)
Comment (gcampo88) [inline src/assets/styles/app.css]: :+1:

File: src/assets/styles/app.css (hunk 3 above)
Comment (gcampo88) [inline src/assets/styles/app.css]: :+1:

File: src/assets/styles/app.css (hunk 4 above)
Comment (gcampo88) [inline src/assets/styles/app.css]: :+1:

File: src/assets/styles/layout.css (hunk 5 above)
Comment (gcampo88) [inline src/assets/styles/layout.css]: :+1:

File: src/assets/styles/layout.css (hunk 6 above)
Comment (gcampo88) [inline src/assets/styles/layout.css]: :+1:

File: src/assets/styles/layout.css (hunk 7 above)
Comment (gcampo88) [inline src/assets/styles/layout.css]: :+1:

File: src/assets/styles/layout.css (hunk 8 above)
Comment (gcampo88) [inline src/assets/styles/layout.css]: :+1:

File: src/assets/styles/layout.css (hunk 9 above)
Comment (gcampo88) [inline src/assets/styles/layout.css]: :+1:

**Pull request merged**